modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -238,7 +238,12 @@ export class {{classname}} extends runtime.BaseAPI {
         {{/isListContainer}}
         {{^isListContainer}}
         if (requestParameters.{{paramName}} !== undefined) {
+            {{#isPrimitiveType}}
             formParams.append('{{baseName}}', requestParameters.{{paramName}} as any);
+            {{/isPrimitiveType}}
+            {{^isPrimitiveType}}
+            formParams.append('{{baseName}}', new Blob([JSON.stringify(requestParameters.{{paramName}})], { type: "application/json", }));
```

Comment:
what if the parameter is e.g. a `Blob`, is it also a primitive type?

Reply:
Yes, it's a primitive. I updated the PR cover letter with test details that includes binary data.